aviatrix/utils.go

```diff
@@ -391,3 +391,20 @@ func sortInterfacesByCustomOrder(interfaces []goaviatrix.EdgeTransitInterface) [
 	})
 	return interfaces
 }
+
+// Sorting function that uses the interface mapping order
+func sortInterfaceMappingByCustomOrder(interfaceMapping []goaviatrix.InterfaceMapping) []goaviatrix.InterfaceMapping {
```

Comment:
We should add tests for this. What happens when you have `eth10`? What happens when the interface names are not in the `interfaceOrder` slice?